v2/pkg/engine/resolve/context.go

```diff
@@ -138,11 +139,29 @@ func (c *Context) SetRateLimiter(limiter RateLimiter) {
 }
 
 func (c *Context) SubgraphErrors() error {
```

Comment:
Is this the only place how router sees errors? How do we expose errors to router in general? I am asking because this interface has not changed, we just return all the errors...

Reply:
The router calls this function in two places, in these cases it does not care about which errors belong to which subgraph, thus I maintained the same behaviour of returning all the errors.